lieferemma_api.proto

```diff
@@ -1,391 +1,408 @@
 
-syntax="proto3";
+syntax = "proto3";
 
 package lieferemma;
 
 import "google/protobuf/timestamp.proto";
 
 message Position {
-    double latitude = 1;
-    double longitude = 2;
+  double latitude = 1;
+  double longitude = 2;
 }
 
 message CustomerInterestRequest {
-    // E.g. center of map
-    Position position_of_interest = 1;
-    // Radius of interest e.g. when viewing map
-    // May be ignored by server when exceeding a certain distance
-    int32 radius_in_meter = 2;
+  // E.g. center of map
+  Position position_of_interest = 1;
+  // Radius of interest e.g. when viewing map
+  // May be ignored by server when exceeding a certain distance
+  int32 radius_in_meter = 2;
 }
 
 enum DeliveryStatus {
-    ON_TOUR = 0;
-    PARKED = 1;
+  // The 0 value is required by protobuf but is not a valid value!
+  DELIVERY_STATUS_UNSPECIFIED = 0;
```

Comment:
I do not understand the the reason for using the namespace in the enum.

Reply:
Ok looks like it is a problem with protobuf not allowing double usage of values in different enums. we will keep it in that case